articles/client-platforms/angularjs.md

```diff
@@ -6,9 +6,9 @@ image: //auth0.com/lib/platforms-collection/img/angular.png
 tags:
   - quickstart
 snippets:
-  dependancies: angularjs/dependancies.html
-  setup: angularjs/app.js
-  use: angularjs/login.js
+  dependancies: client-platforms/angularjs/dependancies
```

Comment:
The issue was that I realized there was name collision in many cases. i.e. server-platforms and server-apis both have node, java, etc. In light of that I thought it best to be consistent with all platforms.

Alternatively, it could have been something like nodejs-api, but it seemed like it didnt make sense to create a new pattern when we already have one.